cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -690,14 +690,14 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
             p("Users with the role 'procedureExecutor' can then run any procedure in the `db.schema` namespace. The procedure will be run using the users own privileges.")
           }
           p("The resulting role should have privileges that only allow executing procedures in the `db.schema` namespace:")
```

Comment:
```suggestion
          p("The resulting role has privileges that only allow executing procedures in the `db.schema` namespace:")
```
I realize that this requires updating in multiple places, so if you're up for it, please go ahead, otherwise disregard. A long term goal is to keep the tense consistently to simple present.

Reply:
I'll accept those you have here, if there are others they won't be done now